pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -20,9 +20,13 @@
 import itertools
 import os
 from datetime import datetime
+from datetime import date
+from typing import Tuple, Any, List
 
 import numpy as np
 import pandas as pd
+from numpy import ndarray, dtype
+from pandas import DataFrame, Series
```

Comment:
This is not neccessary if the entire package is imported.

Reply:
Agree